### DISCUSSION TITLE
ANE: Where to place DLLs required by an ANE when debugging via ADL?

### DISCUSSION THREAD

**Discussion by Adolio** [BowlerHatLLC/vscode-as3mxml#745] [Q&A] [answered]
Hello there!

I would need a bit of help to understand where should/could I place my DLLs required by an ANE when using the AIR debugger (ADL).

Basically, I'm using this extension https://github.com/RossD20Studios/FRESteamWorks and everything works fine if I place the required DLL (`steam_api.dll` (32-bit) and `steam_api64.dll` (64-bit) ) aside the `adl.exe` in my AIR SDK binary folder. 

I noticed later that this post is also confirming my approach: https://stackoverflow.com/questions/9823504/how-to-use-external-dll-in-air-native-extension

Ideally, I would like my DLLs to be placed aside my debug build because the extension also requires a configuration file which needs to be placed at the same location (`steam_appid.txt`). This file could be dependent to the way I build the project (different app id for different configuration).

I tried to play around with the `extdir` option available in the `launch.json` but I didn't manage to make it work as I wanted - the DLLs couldn't be found by the ADL.

Did I miss something?

Thanks a lot in advance for your help

**Comment by joshtynjala** [comment]
I don't really have a lot of experience with ANEs. It might be better to ask in https://github.com/airsdk/Adobe-Runtime-Support/discussions

**Reply by Adolio** [reply, marked as answer]
Thanks, 

I finally manually put them in the SDK aside the adl executable. I didn't find any better solution atm.
